_code-samples/issue-credentials/js/look_up_credentials.js

```diff
@@ -0,0 +1,52 @@
+import { ValueError } from "./errors.js";
+
+const lsfAccepted = 0x00010000;
+
+/**
+ * Looks up Credentials issued by/to a specified XRPL account, optionally
+ * filtering by accepted status. Handles pagination.
+ */
+export async function lookUpCredentials(client, issuer, subject, accepted = "both") {
+  const account = issuer || subject; // Use whichever is specified, issuer if both
+  if (!account) {
+    throw new ValueError("Must specify issuer or subject");
+  }
+
+  accepted = accepted.toLowerCase();
+  if (!["yes", "no", "both"].includes(accepted)) {
+    throw new ValueError("accepted must be 'yes', 'no', or 'both'");
+  }
+
+  const credentials = [];
+  let request = {
+    command: "account_objects",
+    account,
+    type: "credential",
```

Comment:
Specifying ledger_index as validated only fetches the account_objects that are present in the validated ledger.

```suggestion
    type: "credential",
    ledger_index: "validated",
```